app-utils/waybar/autobuild/defines

```diff
@@ -3,8 +3,8 @@ PKGSEC=utils
 PKGDEP="gtkmm-3 jsoncpp libsigc++ fmt wayland spdlog gtk-3 \
     gobject-introspection pulseaudio libnl libappindicator libdbusmenu \
     libmpdclient sndio libevdev libxkbcommon upower pipewire jack \
-    wireplumber playerctl libinput gtk-layer-shell libcava \
-    power-profiles-daemon"
+    wireplumber playerctl libinput gtk-layer-shell libcava"
+PKGRECOM="tuned"
```

Comment:
1. Does tuned replace power-profiles-daemon *in place*?
2. Why RECOM?

Reply:
1. Yes https://github.com/AOSC-Dev/aosc-os-abbs/pull/8272/commits/8ba5c8e3d56ade3107851a381a26262d2558240f
2. user can switch to ppd if they don't like tuned